plugin/src/main/kotlin/com/github/triplet/gradle/play/tasks/PublishApk.kt

```diff
@@ -24,7 +25,9 @@ open class PublishApk : PlayPublishPackageBase() {
     @get:InputFiles
     internal val inputApks by lazy {
         // TODO: If we take a customizable folder, we can fix #233, #227
-        variant.outputs.filterIsInstance<ApkVariantOutput>().map { it.outputFile }
+        variant.outputs.filterIsInstance<ApkVariantOutput>().filter {
+            OutputType.valueOf(it.outputType) == OutputType.MAIN || it.filters.isNotEmpty()
```

Comment:
For context, the other output types are splits in which case they have filters determining which type of split they are. So if the output type is a split but doesn't have filters, it means it's got everything aka a universal APK and we ignore it.